index.php

```diff
@@ -6,4 +6,4 @@
  * @since 1.2.1
  */
 
-?>
+echo '';
```

Comment:
Closing PHP tags have also been something that's recommended against. I've seen varying levels of strictness on that, but part of the thinking is that it helps guard against inadvertent code disclosure?

The section of the standards I've seen is this:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#remove-trailing-spaces